packaging/Dockerfile.centos8-nmstate-dev-nm-1.22-git

```diff
@@ -0,0 +1,48 @@
+# This Dockerfile is based on the recommendations provided in the
+# Fedora official repository
+# (https://hub.docker.com/r/fedora/systemd-systemd/).
+# It enables systemd to be operational.
+FROM docker.io/library/centos:8
```

Comment:
So simplify things. Can you just use

```
FROM docker.io/nmstate/centos8-nmstate-dev
RUN dnf copr enable nmstate/ovs-el8 -y && \
    dnf copr enable networkmanager/NetworkManager-1.22-git -y && \
    dnf upgrade NetworkManager -y && \
    dnf clean all

Reply:
This is not possible because the issue is on the rpmdb. I am getting the GPG Check error when building that dockfile locally.